### PR TITLE
feat: Get services by country, city

### PR DESCRIPTION
**Pull request by kevinjanada**
## JIRA Link
 https://blocksphere2020.atlassian.net/browse/DBIO-506?atlOrigin=eyJpIjoiY2E1YjAxNThjMTYxNDU4ZThkMjYwZDVhYjg0MDY3NTMiLCJwIjoiaiJ9
 ## CHANGELOG
 - GET services by country and city.
  Data is retrieved from elasticsearch database,
  where DeBio indexer stores indexed blockchain data

### CODE REVIEW

**Review by bio-sync — APPROVED** [review]

File: src/substrate-indexed-data/services/service.controller.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Controller, Get, Param, Req } from '@nestjs/common';
+import { Request } from 'express';
+import { ServiceService } from './service.service';
+
+@Controller('services')
+export class ServiceController {
+  constructor(readonly serviceService: ServiceService) {}
```
Comment (agustinustheo) [inline src/substrate-indexed-data/services/service.controller.ts:7]: Service Service?

File: src/substrate-indexed-data/services/service.service.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { ElasticsearchService } from '@nestjs/elasticsearch';
+
+@Injectable()
+export class ServiceService {
+  constructor(private readonly elasticsearchService: ElasticsearchService) {}
+
+  async getByCountryCity(country: string, city: string) {
+    console.log(country);
```
Comment (agustinustheo) [inline src/substrate-indexed-data/services/service.service.ts:9]: Console log mending dihapus dulu

File: src/substrate-indexed-data/services/service.controller.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  @Get(':country/:city')
+  async findByCountryCity(@Param() params): Promise<any> {
+    const services = await this.serviceService.getByCountryCity(params.country, params.city);
```
Comment (agustinustheo) [inline src/substrate-indexed-data/services/service.controller.ts:11]: kenapa gak langsung return aja?

Comment (kevinjanada) [reply src/substrate-indexed-data/services/service.controller.ts:11]: eh return await bisa ya?

File: src/substrate-indexed-data/services/service.controller.ts (hunk 1 above)
Comment (kevinjanada) [reply src/substrate-indexed-data/services/service.controller.ts:7]: Any suggestion for a better name?

**Pull request merged**